conda-forge.yml

```
@@ -1,10 +1,18 @@
 bot:
```

We can store artifacts at the end of CI for local testing if that is helpful. To do this, would add this to the config and [re-render]( https://conda-forge.org/docs/maintainer/updating_pkgs/#rerendering-feedstocks )

```suggestion
azure:
  store_build_artifacts: true
bot:
```